### PR TITLE
[AdherentMessage] Sync Referent message filters

### PR DESCRIPTION
**Pull request by ottaviano**

### CODE REVIEW

File: src/Referent/ManagedUsersFilter.php [hunk 1]
```diff
@@ -273,4 +306,41 @@ public function includeCitizenProject(): bool
     {
         return $this->includeCP;
     }
+
+    public function serialize()
```
Comment (JeremieSamson) [inline src/Referent/ManagedUsersFilter.php]: typehint ?

File: src/Referent/ManagedUsersFilter.php [hunk 2]
```diff
@@ -273,4 +306,41 @@ public function includeCitizenProject(): bool
...
+            $this->queryGender,
+            $this->queryLastName,
+            $this->queryFirstName,
+            $this->queryAgeMinimum,
+            $this->queryAgeMaximum,
+            $this->queryInterests,
+        ]);
+    }
+
+    public function unserialize($serialized)
```
Comment (JeremieSamson) [inline src/Referent/ManagedUsersFilter.php]: typehint ?

File: src/Controller/EnMarche/MessageController.php [hunk 3]
```diff
@@ -131,15 +135,22 @@ public function updateMessageAction(
...
     /**
      * @Route("/{uuid}/visualiser", name="app_message_preview", methods={"GET"})
-     * @Security("message.isSynchronized() && is_granted('IS_AUTHOR_OF', message)")
+     *
+     * @Security("is_granted('IS_AUTHOR_OF', message)")
      */
     public function previewMessageAction(string $prefix, AbstractAdherentMessage $message): Response
     {
+        if (!$message->isSynchronized()) {
+            throw new BadRequestHttpException('Message preview is not yet ready.');
```
Comment (thomasbisignani) [inline src/Controller/EnMarche/MessageController.php]: `Message preview is not ready yet.`

Comment (ottaviano) [reply src/Controller/EnMarche/MessageController.php]: fixed, thanks

File: src/Referent/ManagedUsersFilter.php (hunk 1 above)
Comment (ottaviano) [reply src/Referent/ManagedUsersFilter.php]: it is `\Serializable` interface signatures

File: src/Referent/ManagedUsersFilter.php (hunk 2 above)
Comment (ottaviano) [reply src/Referent/ManagedUsersFilter.php]: same

**Pull request merged**